utilities/utils/response.go

```diff
@@ -92,7 +92,12 @@ func getInstanceHostMapData(event *revents.Event, client *client.Client, cache *
 	}
 	dockerIP, err := getIP(inspect, cache)
 	if err != nil {
-		return map[string]interface{}{}, errors.Wrap(err, constants.GetInstanceHostMapDataError+"failed to get ip of the container")
+		if val, ok := inspect.Config.Labels[constants.RancherIPLabel]; ok && val != "" {
```

Comment:
I dont think this is the right thing to  do. I think the right thing to do is to check the state of the container in lookUpIP and if the container is not running, just dont return an IP if the container is stopped.